### ISSUE TITLE
Use FluentValidation instead of FluentValidation.AspNetCore

### ISSUE THREAD

**Issue by cremor** [micro-elements/MicroElements.Swashbuckle.FluentValidation#111]
Continuation of #97 

FluentValidation.AspNetCore support is ending, see FluentValidation/FluentValidation#1959
The workaround mentioned in the readme and in #97 is showing compiler warnings because both `IValidatorFactory` and `ServiceProviderValidatorFactory` are marked as obsolete in FluentValidation, see FluentValidation/FluentValidation#1961

I suggest to modify this (very useful!) library to work without FluentValidation.AspNetCore by default.
Until this is done an updated workaround that doesn't raise compiler warnings would be great.

**Comment by petriashev** [comment]
Yes I heard about these changes. Will try to adapt code for new DI needs.

**Comment by petriashev** [comment]
Please check 6.0.0-beta.1

**Comment by cremor** [comment]
6.0.0-beta.1 seems to work fine, thanks.

**Comment by cremor** [comment]
@petriashev  When will v6 be released?

**Comment by petriashev** [comment]
> @petriashev When will v6 be released?

I was on vacation and before that was busy on 140%. 
It's not my main project. Ping me sometime!
